### PR TITLE
[16.0][MIG] purchase_landed_cost: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration from v13 to v16
- Squash administrative commits
- Depends on https://github.com/OCA/purchase-workflow/pull/2221 that solves CI error tests

@Tecnativa
TT46594

@pilarvargas-tecnativa @pedrobaeza

### CODE REVIEW

**Comment by carolinafernandez-tecnativa** [comment]
Checked CI and the problem is not from the module migrated.

**Comment by carolinafernandez-tecnativa** [comment]
@pilarvargas-tecnativa @pedrobaeza rebased and ready for review

**Review by pilarvargas-tecnativa — APPROVED** [review]

File: purchase_landed_cost/wizard/picking_import_wizard.py [hunk 1]
```diff
@@ -18,14 +18,13 @@ def default_get(self, field_list):
             for line in distribution.cost_lines:
                 if line.picking_id in pickings:
                     continue
-                if all(x in moves for x in line.picking_id.move_lines):
+                if all(x in moves for x in line.picking_id.move_line_ids):
```
Comment (pedrobaeza) [inline purchase_landed_cost/wizard/picking_import_wizard.py]:
```suggestion
                if all(x in moves for x in line.picking_id.move_ids):
```

File: purchase_landed_cost/wizard/picking_import_wizard.py [hunk 2]
```diff
@@ -50,8 +49,8 @@ def action_import_picking(self):
             self.env.context["active_id"]
         )
         previous_moves = distribution.mapped("cost_lines.move_id")
-        for move in self.mapped("pickings.move_lines"):
-            if move not in previous_moves:
+        for move in self.mapped("pickings.move_line_ids"):
```
Comment (pedrobaeza) [inline purchase_landed_cost/wizard/picking_import_wizard.py]:
```suggestion
        for move in self.mapped("pickings.move_ids"):
```

and subsequent changes

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
/ocabot migration purchase_landed_cost

File: purchase_landed_cost/wizard/picking_import_wizard.py (hunk 2 above)
Comment (carolinafernandez-tecnativa) [reply purchase_landed_cost/wizard/picking_import_wizard.py]: fix it, thanks!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**